src/plugin.ts

```diff
@@ -27,10 +27,7 @@ export default class StackOutputPlugin {
   }
 
   get stackName () {
-    return util.format('%s-%s',
-      this.serverless.service.getServiceName(),
-      this.serverless.getProvider('aws').getStage()
-    )
+    return this.serverless.getProvider('aws').naming.getStackName()
```

Comment:
I've seen this same behavior, too. Out of curiosity, why not just read directly from the property (e.g. `this.serverless.getProvider('aws').stackName`) and declare that in the typings file?

This this value be populated if the stackName property is *not* set in serverless.yml? It's possible that we need to fall back to the defaulted value here?

Reply:
nm; It looks like it is fine even if stackName is not specified in serverless.yml